gluon/_compat.py

```diff
@@ -0,0 +1,106 @@
+import sys
```

Comment:
because pydal is a separate project and it may change it.

2016-06-07 16:00 GMT+02:00 BuhtigithuB <notifications@github.com>:

> In gluon/_compat.py
> <https://github.com/web2py/web2py/pull/1339#discussion_r66075531>:
>
> > @@ -0,0 +1,106 @@
> > +import sys
>
> Why are we duplicate this file?
>
>
> https://github.com/web2py/pydal/blob/b5f91ba4837da33f1f3036fa5a821a28dee68693/pydal/_compat.py
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/web2py/web2py/pull/1339/files/48e10a379335c215e97af4cbf4ea97cee97671c3..aa2bf4134ee9d1c5351d449bff8592785eb1df47#r66075531>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAHdB7s7F6uyDzVWkJpk4xFUjU3bS_zbks5qJXl2gaJpZM4Iohel>
> .
>